scripts/people.js

```diff
@@ -0,0 +1,29 @@
+// Commands:
+//    hubot people - List all GB members
+//    hubot people [query] - Search people by any field
+
+var hq = require('../lib/headquarters');
+
+module.exports = function(robot) {
+  robot.respond(/people$/i, function(msg) {
+    hq.member.all().then(function(members) {
+      msg.send(parsedMembers(members));
+    });
+  });
+
+  robot.respond(/people (.*)/i, function(msg) {
+    var query = msg.match[1];
+    var members = hq.member.search(query).then(function(members) {
+      msg.send(parsedMembers(members));
+    });
+  })
+};
+
+function parsedMembers(members) {
+  return members.map(function(member) {
+    return member.name + " - " +
```

Comment:
What about all the other fields? Should it be possible to fetch them optionally, or just dump all the data?